cluster/gce/gci/node.yaml

```diff
@@ -117,6 +117,12 @@ write_files:
       [Install]
       WantedBy=multi-user.target
 
+  - path: /etc/modprobe.d/sunrpc.conf
```

Comment:
this is always present, correct? Any chance there is a configuration without this?

Reply:
/etc/modprobe.d always exists in the COS and ubuntu images supported (as I understand) by gci. sunrpc.conf does not currently exist anywhere I've looked, so we're creating it with this PR.